builders/manage/dapp/services-payment.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Services Payment
+description: Appchains deployed through Tanssi benefit from block production services provided by a set of node operators, which are compensated with Tanssi tokens.
+---
+
+# Services Payment
+
+## Introduction {: #introduction }
+
+As presented in the [Block Production as a Service](/learn/tanssi/technical-features#services-payments){target=_\blank} article, there are two main costs associated that the appchain governor must cover: 
+
+- **Block producers assignment** - for the assignment of block producers by the Tanssi protocol, which happens once per session
+- **Block production** - for each block that is produced on the appchain's behalf
+
+In this guide, you'll learn how to use the [Tanssi dApp](https://apps.tanssi.network){target=\_blank} to top-up your account and keep your appchain's liveness.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+- A Tanssi appchain (Snap or Dedicated)
+- The account you used when registering the appchain, imported in any of the [supported wallets](/builders/deploy/dapp/#supported-wallets){target=\_blank}
+
+If you're unsure what your registration account is, you can query it directly from the Tanssi orchestrator chain, which keeps records of every registered appchain. To do so, head to the **Chain state** section on the Polkadot.js Apps connected to the orchestrator chain for [snap appchains](https://polkadot.js.org/apps/?rpc=wss://fraa-flashbox-2607-rpc.a.stagenet.tanssi.network#/chainstate){target=\_blank} or [dedicated appchains](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}, and take the following steps:
+
+1. Select the **registrar** storage module
+2. Select **registrarDeposit**
+3. Insert you appchain id
+4. Press **+** icon
+
+You'll see the registration account at the bottom.
+
+![Locating your registration account](/images/builders/manage/dapp/services-payment/services-payment-1.webp)
+
+## Topping-Up Your Appchain's Credits {: #topping-up }
+
+Following a pay-as-you-go model, the appchains must have funds allocated to pay for the services, which, in time, will be deducted by the Tanssi protocol upon every session change for the cost of block producer assignment and every block for the cost of producing the block.
+
+!!! note
```

Comment:
This has to be in a note?

Reply:
Revisiting this paragraph, I think it sets the importance of having enough funds just before explaining how to check & refill the credits. I wouldn't put it in a note.
But, I would remove the next lines about how the 20% goes to the treasury, and move it to the payment article in the learn section. wdyt?